src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
```

Comment:
This is just the prototype, but next steps might be to add the navbar to this page, and update the navbar on the other pages so that they have the activity feed accessible.